Makefile.third

```diff
@@ -197,8 +202,10 @@ endif
 $(DJVULIBRE_LIB): $(JPEG_LIB) $(THIRDPARTY_DIR)/djvulibre/*.*
 	install -d $(DJVULIBRE_BUILD_DIR)
 	cd $(DJVULIBRE_BUILD_DIR) && \
-		$(CMAKE) $(CMAKE_FLAGS) -DCC="$(CC)" -DCXX="$(CXX)" -DCFLAGS="$(CFLAGS)" \
-		-DCXXFLAGS="$(CXXFLAGS)" -DLDFLAGS="$(LDFLAGS)" \
+		$(CMAKE) $(CMAKE_FLAGS) -DCC="$(CC)" -DCXX="$(CXX)" \
+		-DCFLAGS="$(CFLAGS)" \
+		-DCXXFLAGS="$(CXXFLAGS)" \
+		-DLDFLAGS="$(LDFLAGS) -Wl,-rpath,'$(ORIGIN_CMAKE_TO_AUTOCFG)'" \
```

Comment:
I don't think the whole `ORIGIN_CMAKE_TO_AUTOCFG` hack is even needed anymore, e.g. this works fine:
```diff
 Makefile.third | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git i/Makefile.third w/Makefile.third
index 27850c15..2e45cba5 100644
--- i/Makefile.third
+++ w/Makefile.third
@@ -101,7 +101,7 @@ $(PNG_LIB): $(ZLIB) $(THIRDPARTY_DIR)/libpng/*.*
 	cd $(PNG_BUILD_DIR) && \
 		$(CMAKE) $(CMAKE_FLAGS) -DCC="$(CC)" -DCXX="$(CXX)" -DCHOST="$(CHOST)" \
 		-DCPPFLAGS="$(CFLAGS) -I$(ZLIB_DIR)" \
-		-DLDFLAGS="$(LDFLAGS) -L$(ZLIB_DIR) -Wl,-rpath,'$(ORIGIN_CMAKE_TO_AUTOCFG)'" \
+		-DLDFLAGS="$(LDFLAGS) -L$(ZLIB_DIR)" \
 		$(CURDIR)/$(THIRDPARTY_DIR)/libpng && \
 		$(CMAKE_MAKE_PROGRAM) $(CMAKE_MAKE_PROGRAM_FLAGS)
 	cp -fL $(PNG_DIR)/.libs/$(notdir $(PNG_LIB)) $@
```
```bash
▸ readelf -d build/x86_64-pc-linux-gnu/libs/libpng16.so.16 | grep 'File:\|NEEDED\|RUNPATH'
 0x0000000000000001 (NEEDED)             Shared library: [libz.so.1]
 0x0000000000000001 (NEEDED)             Shared library: [libm.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libmvec.so.1]
 0x0000000000000001 (NEEDED)             Shared library: [libc.so.6]
 0x000000000000001d (RUNPATH)            Library runpath: [$ORIGIN:$ORIGIN/libs]
```

Reply:
Yep, it'll be gone ;).